fili-core/src/main/java/com/yahoo/bard/webservice/table/PhysicalTable.java

```diff
@@ -39,12 +45,11 @@
     DateTime getTableAlignment();
 
     /**
-     * Get the schema for this physical table.
-     * Schemas contain granularity and column definitions.
+     * Getter for all possible column available intervals.
```

Comment:
Drop `possible`. There _could_ be data from the year 5billion, so this isn't "possible", but rather a "Getter for all of the available intervals for the columns in this table"